packages/dashboard/src/components/ConfirmNetworkChange.tsx

```diff
@@ -30,7 +30,7 @@ function ConfirmNetworkChanged({
     </div>
   );
 
-  const confirmButton = <Button text="Confirm" onClick={confirm} />;
+  const confirmButton = <Button text="Confirm" disabled={false} onClick={confirm} />;
```

Comment:
I don't think we should include this if we're just passing false

```suggestion
  const confirmButton = <Button text="Confirm" onClick={confirm} />;
```